docker/compute_worker/compute_worker.py

```diff
@@ -395,7 +445,18 @@ def push_scores(self):
         logger.info(resp)
         logger.info(str(resp.content))
 
-    def push_result(self):
+    def push_output(self):
+        """Output is pushed at the end of both prediction and scoring steps."""
+        # V1.5 compatibility, write program statuses to metadata file
+        prog_status = {
+            'exitCode': self.program_exit_code,
+            'ingestionExitCode': self.ingestion_program_exit_code,
+            'elapsedTime': self.program_elapsed_time
+        }
+
+        with open(os.path.join(self.output_dir, 'metadata'), 'w') as f:
```

Comment:
TODO for Eric: if this file already exists in output, from participant/organizer's program... raise some kind of `SubmissionException`!